### ISSUE TITLE
Add timestamps to confirmed and spent block columns

### ISSUE THREAD

**Issue by BrandtH22** [xch-dev/sage#38]
The wallet currently displays the confirmed and/or spent block height for coins/tokens, it would be much more user friendly to instead display the timestamp of the transaction block and have the block height (and potentially transaction ID) in a tooltip for the coin/token

**Comment by Rigidity** [comment]
Good point, the wallet protocol doesn't provide this data though, so I'm not sure what the best way to get it would be.

**Comment by Rigidity** [comment]
The wallet protocol doesn't have a message for this, but it turns out the peer protocol does. You can request the block header for the spent and confirmed heights and get the timestamp out of the foliage block info. I'll make a queue to request all of the block timestamps for the coin records in the wallet.
